.profile.d/heroku-metrics-daemon.sh

```diff
@@ -32,13 +32,13 @@ echo "agentmon setup took ${ELAPSEDTIME} seconds"
 
 AGENTMON_FLAGS=()
 
-if [ -f pom.xml ] || [ -f build.gradle ] || [ -d .jdk ]; then
-    export JAVA_TOOL_OPTIONS="-javaagent:bin/heroku-metrics-agent.jar ${JAVA_TOOL_OPTIONS}"
-    AGENTMON_FLAGS+=("-prom-url=http://localhost:${HEROKU_METRICS_PROM_PORT}${HEROKU_METRICS_PROM_ENDPOINT}")
-elif [ -f build.sbt ]; then
+if [ -f build.sbt ]; then
```

Comment:
[4:43 PM] <freeformz> @joe Can you please change all of those to `[[ ]]` instead of `[ ]` ?
[4:44 PM] <joe> sure, but why?
[4:44 PM] <freeformz> consistency

https://heroku.slack.com/archives/C1R38F23T/p1492127013270151